src/components/SourceTabs.js

```diff
@@ -233,7 +233,7 @@ const SourceTabs = React.createClass({
 
   renderTab(source) {
     const { selectedSource, selectSource, closeTab } = this.props;
-    const filename = getRawSourceURL(getFilename(source.toJS()));
+    const filename = getURL(source.get("url")).filename;
```

Comment:
I think we still want to call `getRawSourceURL` which removes the ":formatted" suffix from URLs we pretty print